src/date-picker/date-picker.scss

```diff
@@ -62,58 +63,220 @@
   > div {
     white-space: nowrap;
     overflow: hidden;
-    width: $iui-xl;
+    width: $iui-date-picker-cell-width;
     text-align: center;
   }
 }
 
 @mixin iui-calendar-week {
   white-space: nowrap;
-  height: $iui-baseline * 3;
+  line-height: $iui-date-picker-cell-height;
   display: flex;
-  align-items: center;
-  justify-content: space-evenly;
   padding: 0 $iui-s;
+  margin-bottom: $iui-xxs;
 }
 
-@mixin iui-calendar-day {
+//#region Helper mixins
+@mixin iui-calendar-day-base {
   @include iui-focus;
   cursor: pointer;
-  display: inline-block;
-  width: $iui-baseline * 3;
-  line-height: $iui-baseline * 3;
+  text-align: center;
+  width: $iui-date-picker-cell-width;
+  height: $iui-date-picker-cell-height;
   border-radius: $iui-border-radius;
+  font-variant-numeric: tabular-nums;
```

Comment:
What is the benefit of having this? Because I don't see any changes when toggling it.

Reply:
Try hovering over the day and see numbers moving horizontally while changing font-weight

Reply:
This is how I found out about this CSS property: https://twitter.com/javan/status/1486059026064584711